contrib/autopilot/tasks.py

```diff
@@ -104,6 +108,10 @@
                               "http://opus.nlpl.eu/download.php?f=SETIMES2/en-tr.txt.zip",
                               "544cec8a631f7820afab6a05451c13a7",
                               ARCHIVE_ZIP),
+    "paracrawl_release1_en_de": RawFile("Paracrawl",
```

Comment:
Tiny note: the descriptions generally include some type of version information as well.  We could use "Paracrawl r1".